### PR TITLE
Add master options to snmpd.conf

### PR DESCRIPTION
**Pull request by coreone**
* Add master and agentx options to snmpd.conf.erb
* Fix typo in CONTRIBUTING

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -432,6 +429,29 @@ Default: true
 Adds the smuxpeer directive to the snmpd.conf file to allow net-snmp to talk with Dell's OpenManage.
 Default: false
 
+##### `master`
+Allow setting the *master* option, typically to enable AgentX registrations.
+Default: agentx
```
Comment (razorsedge) [inline README.markdown]: Should be "Default: false" based on the actual value in params.pp. And should probably be worded differently.

File: manifests/init.pp [hunk 2]
```diff
@@ -315,8 +340,15 @@
...
   $openmanage_enable            = $snmp::params::openmanage_enable,
+  $master                       = $snmp::params::master,
+  $agentx_perms                 = $snmp::params::agentx_perms,
+  $agentx_ping_interval         = $snmp::params::agentx_ping_interval,
+  $agentx_socket                = $snmp::params::agentx_socket,
+  $agentx_timeout               = $snmp::params::agentx_timeout,
+  $agentx_retries               = $snmp::params::agentx_retries,
 ) inherits snmp::params {
   # Validate our booleans
+  validate_bool($master)
```
Comment (razorsedge) [inline manifests/init.pp]: Can we validate the other inputs in some way?

File: manifests/params.pp [hunk 3]
```diff
@@ -436,6 +478,7 @@
       $client_package_name      = 'snmp'
       $client_config            = '/etc/snmp/snmp.conf'
 
+      $trap_service_name        = 'snmptrapd'
```
Comment (razorsedge) [inline manifests/params.pp]: Why is this here?  It is causing all the Travis tests to fail.

File: manifests/init.pp (hunk 2 above)
Comment (coreone) [inline manifests/init.pp]: * Readme fixed
* Added validation / new defaults for two of the AgentX options that had defaults listed in the man page.  The other options have no defaults (i.e. undef)
* Fixed duplicate variable from bad rebase

Comment (coreone) [inline manifests/init.pp]: @razorsedge is anything else needed on this?  Additionally, should close #96
